backend/src/db/providers/clientCourseProvider.ts

```diff
@@ -1,7 +1,9 @@
+import { ObjectId } from 'mongoose';
```

Comment:
Empty line! Mongoose is a library and it should stand separately.

Reply:
done